selenium-standalone-chrome/Dockerfile

```diff
@@ -0,0 +1,10 @@
+#ddev-generated
+
+FROM seleniarm/standalone-chromium:4.1.4-20220429
+ENV CAROOT=/mnt/ddev-global-cache/mkcert
+
+USER root
+RUN apt update >/dev/null && apt install -y iputils-ping less mkcert vim >/dev/null
```

Comment:
I guess this could be done with a curl install in post-start hook, but it's less intrusive here. But that would get rid of the whole Dockerfile.